types/resolve-pkg/resolve-pkg-tests.ts

```diff
@@ -0,0 +1,5 @@
+import * as rpkg from 'resolve-pkg';
```

Comment:
I think this should be `import rpkg = require('resolve-pkg');` if the result needs to be callable.